libs/designsystem/card/src/card.component.scss

```
@@ -59,18 +60,18 @@
 
   &.outlined {
     background-color: transparent;
-    outline: 1px solid utils.get-color('medium');
+    border: 1px solid utils.get-color('medium');
```

I guess we should allow a custom border in outlined cards as well? 🤔 
```suggestion
    border: border: var(--kirby-card-border, 1px solid utils.get-color('medium'));
```